docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -2041,6 +2043,138 @@ The third example will autoregister the device and assign the role staff to each
   scope = NodeInfoForAutoReg
   role = staff
 
+EXAMPLE: autoregister all Windows devices and assign them the role default.
+
+  [Windows_AutoReg]
+  filter = fingerbank_info.device_hierarchy_names
+  operator = includes
+  value = Windows
+
+  [AUTOREG_WINDOWS:Windows_AutoReg]
+  scope = AutoRegister
+  role = default
+
+  [WINDOWS_ROLE:Windows_devices]
+  scope = NodeInfoForAutoReg
+  role = default
+
+
+EXAMPLE: filter a MAC address and reject it by assigning the role REJECT.
+
+  [MAC]
+  filter = node_info.mac
+  operator = match
+  value = AA:BB:CC:DD:EE:FF
+
+  [SSID_Open]
+  filter = ssid
+  operator = is
+  value = Open
+
+  [REJECT:MAC&SSID_Open]
+  scope = RegistrationRole
+  role = REJECT
+
+EXAMPLE: autoregister devices using dot1x/EAP over wired or wireless and that are not pending
+
+  [not_pending]
+  filter = node_info.status
+  operator = is_not
+  value = pending
```

Comment:
Not sure that the pending pending state is available in the vlan filters since it's a a portal session attribute.

Reply:
check with @julsemaan

Reply:
Pending is still a valid node state when using provisioning